dali/test/tensor_list_wrapper.h

```diff
@@ -73,6 +80,19 @@ class TensorListWrapper {
     return *gpu_;
   }
 
+  template <typename Backend>
+  std::unique_ptr<TensorList<Backend>> CopyTo() const {
+    std::unique_ptr<TensorList<Backend>> result(new TensorList<Backend>());
+    DALI_ENFORCE(has_cpu() != has_gpu(), "Should contain TensorList from exactly one backend");
```

Comment:
I think it can't happen. Plain `assert` would do, if any at all.

Reply:
It can be empty though, and it checks against those two situations. I just want to make sure, that if something is changed later and it can carry both cpu and gpu TL, somebody have to decide which one to copy here.